src/main/java/org/jboss/forge/arquillian/command/algeron/consumer/PactAlgeronConsumerTestSetup.java

```diff
@@ -33,6 +33,7 @@ public JavaClassSource updateTest(JavaClassSource test, String consumer, String
         annotateTestClass(test, consumer, provider);
         createContractFragment(test, fragmentName);
         createTestMethod(test, consumer, provider, fragmentName);
+        removeTestMethod("should_be_deployed", test);
```

Comment:
No it's not possible as we don't have Deployment method. we have test method named should_be_deployed()